### PR TITLE
Add Documentation on Load Balancing in UV

### PR DESCRIPTION
**Pull request by Notplayingallday383**
Hi there, So this PR is a small one and basically adds a bit of documentation of how to load balance your UV instance. Feel free to revise and modify this.

### CODE REVIEW

**Comment by proudparrot2** [comment]
The now.gg bypass is very unstable rn

**Comment by Notplayingallday383** [comment]
> The now.gg bypass is very unstable rn

I know I just replaced the url from actual nowgg to the bypass cus trying to play nowgg without it on UV is just usless lol

File: README.md [hunk 1]
```diff
@@ -47,14 +53,14 @@ If you're too lazy to do either of the above, you can install an outdated and un
...
 
 ## Deployment
 
-This repository can be built, but it can't be deployed to any services. See [Ultraviolet-App](https://github.com/titaniumnetwork-dev/Ultraviolet-App) for a complete and customizable Ultraviolet setup that you can deploy.
+This repository can be built for ultraviolet scripts that you can add to your own website, but this repository can't be deployed to any services. See [Ultraviolet-App](https://github.com/titaniumnetwork-dev/Ultraviolet-App) for a complete and customizable Ultraviolet setup that you can deploy.
 
 ## Documentation
 
-Documentation can be found in the [wiki](https://github.com/titaniumnetwork-dev/Ultraviolet/wiki).
+Documentation for can be found in the [wiki](https://github.com/titaniumnetwork-dev/Ultraviolet/wiki).
```
Comment (9rh) [inline README.md]: What was the point of editing this?

Comment (Notplayingallday383) [reply README.md]: oops I made a mistake my bad

**Comment by Notplayingallday383** [comment]
I dont know why github isn't saying I updated it but I fixed the spelling mistake

**Review by VillainsRule — CHANGES_REQUESTED** [review]
this pr sux

**Pull request closed without merging**